cypress-custom/integration/fee.test.ts

```diff
@@ -16,23 +16,30 @@ function _getLocalStorage(): Cypress.Chainable<Storage> {
   return cy.window().then(window => window.localStorage)
 }
 
+function _getChainFeeStorage(networkKey: string): Cypress.Chainable {
+  return (
+    _getLocalStorage()
+      .its(FEE_QUOTES_LOCAL_STORAGE_KEY)
+      // To properly return this we need .should and an expectation
+      .should(feeQuotesStorage => {
+        expect(JSON.parse(feeQuotesStorage)).to.have.property(networkKey)
+      })
+      .then(fee => JSON.parse(fee)[networkKey])
+  )
+}
+
 function _assertFeeFetched(token: string): void {
-  _getLocalStorage()
-    .its(FEE_QUOTES_LOCAL_STORAGE_KEY)
-    .should(feeQuotesStorage => {
-      // THEN: There is fee information for Rinkeby and token
-      const feeQuoteData = JSON.parse(feeQuotesStorage)
-      expect(feeQuoteData).to.exist
-      expect(feeQuoteData).to.have.property(RINKEBY)
-      expect(feeQuoteData[RINKEBY]).to.have.property(token)
-
-      // THEN: The quote has the expected information
-      const fee = feeQuoteData[RINKEBY][token].fee
-      _assertFeeData(fee)
-    })
+  _getChainFeeStorage(RINKEBY).then(feeQuoteData => {
+    expect(feeQuoteData).to.exist
+    expect(feeQuoteData).to.have.property(token)
+
+    // THEN: The quote has the expected information
+    const fee = feeQuoteData[token].fee
+    _assertFeeData(fee)
+  })
 }
 
-describe('Fee endpoint', () => {
+xdescribe('Fee endpoint', () => {
```

Comment:
Is this intentional? Is `xdescribe` a valid command?

Reply:
yeah, it ignores tests